.storybook/preview.js

```diff
@@ -2,6 +2,11 @@ import React from 'react'
 import { addDecorator } from '@storybook/react'
 import { BrandingProvider, ThemeProvider, GlobalStyle } from '../src'
 
+const rootElementStyle = document.documentElement.style
+
+rootElementStyle.setProperty('--etvas-brand-color', '#0040E3')
+rootElementStyle.setProperty('--etvas-brand-color-light', '#5585FF')
+
```

Comment:
Can't you just use [this](https://github.com/etvascom/etvas-kit/blob/4bfd139e8f7e3be818e5059a633f87c7a233f8c9/src/providers/BrandingService.js#L88)?